backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala

```diff
@@ -171,6 +171,8 @@ object CHExpressionUtil {
     RPAD -> StringRPadValidator(),
     DATE_FORMAT -> DateFormatClassValidator(),
     DECODE -> EncodeDecodeValidator(),
-    ENCODE -> EncodeDecodeValidator()
+    ENCODE -> EncodeDecodeValidator(),
+    ARRAY_EXCEPT -> DefaultValidator(),
+    ARRAY_REPEAT -> DefaultValidator()
```

Comment:
cc @taiyang-li, I directly changed this for CH backend to let these two unsupported functions fall back.